yt_dlp/postprocessor/embedthumbnail.py

```diff
@@ -90,12 +90,16 @@ def run(self, info):
 
         success = True
         if info['ext'] == 'mp3':
-            options = [
-                '-c', 'copy', '-map', '0:0', '-map', '1:0', '-write_id3v1', '1', '-id3v2_version', '3',
-                '-metadata:s:v', 'title="Album cover"', '-metadata:s:v', 'comment=Cover (front)']
-
-            self._report_run('ffmpeg', filename)
-            self.run_ffmpeg_multiple_files([filename, thumbnail_filename], temp_filename, options)
+            # Using ffmpeg to embed the thumbnail in mp3 files is messing up lyrics
```

Comment:
fallback to ffmpeg if mutagen is not available. See how we do it for mp4

Reply:
i changed it 
please have a look at it